Tasks/GradlePreview/Gradle.ps1

```diff
@@ -73,30 +97,23 @@ if($publishJUnitResultsFromAntBuild)
     }
     else
     {
-        Write-Verbose "Calling Publish-TestResults"
+        Write-Verbose "Calling Publish-TestResults" -Verbose
         Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext
     }    
 }
 else
 {
-    Write-Verbose "Option to publish JUnit Test results produced by Gradle build was not selected and is being skipped."
+    Write-Verbose "Option to publish JUnit Test results produced by Gradle build was not selected and is being skipped." -Verbose
 }
 
-if($publishCodeCoverageResults)
+# check if code coverage has been enabled
+if($isCoverageEnabled)
 {
-   # Publish Code Coverage Files
-   $codeCoverageFiles = Find-Files -SearchPattern $additionalCodeCoverageFiles
-   
-   if(-not $codeCoverageFiles)
-   {
-      Write-Warning "No code coverage files matching pattern '$additionalCodeCoverageFiles' were found."  
-   }
-   
-   Publish-CodeCoverage -CodeCoverageTool $codeCoverageTool -SummaryFileLocation $summaryFileLocation -ReportDirectory $reportDirectory -AdditionalCodeCoverageFiles $codeCoverageFiles -Context $distributedTaskContext    
+   Publish-CodeCoverage -CodeCoverageTool $codeCoverageTool -SummaryFileLocation $summaryFile -ReportDirectory $reportDirectory -Context $distributedTaskContext    
 }
 else
 {
-    Write-Verbose "Option to publish CodeCoverage results produced by Gradle build was not selected and is being skipped."
+    Write-Verbose "Option to publish CodeCoverage results produced by Gradle build was not selected and is being skipped." -Verbose
```

Comment:
I dont think this message makes sense. Now the option is not to publish cc but to enable.